package.json

```diff
@@ -51,10 +51,10 @@
     "admin-lte": "^3.2.0",
     "bootstrap": "^4.6.1",
     "chart.js": "^4.0.1",
-    "chartjs-adapter-dayjs-3": "^1.2.3",
+    "chartjs-adapter-dayjs-4": "^1.0.4",
     "country-code-emoji": "^2.3.0",
     "dayjs": "^1.11.6",
-    "felte": "1.2.6",
+    "felte": "1.2.14",
```

Comment:
This upgrade will break my pulldowns in forms. :( Not sure why, and not much time to investigate. Took me a couple of months of testing... but the version 1.2.6 is the last working version for me.
Please revert.

Reply:
I will downgrade it. However, do you know which form fields exactly were breaking? In my local, it seems to work.

Reply:
When you try to add a new area. When the form is open, and you have not entered anything, press the save button. This will trigger a warning about required fields.

Now you change the first pulldown, turns green. Than change a second pulldown, turns green. But than the first pulldown turns red again. Somehow all the pulldowns are resetting each other.

And you are not able to save the form at all. This only happens when there is an error with the form when submitting. I could not find why, so I stick with the 1.2.6 version.

Reply:
I use the `--force` param with the `npm` commands